modules/RTC/TraceablePeerConnection.js

```diff
@@ -2824,6 +2824,11 @@ TraceablePeerConnection.prototype._createOfferOrAnswer = function(
                     .filter(caps => caps.mimeType.toLowerCase() !== `${MediaType.VIDEO}/${mimeType}`);
             }
 
+            if (browser.isChromiumBased() && browser.isVersionEqualTo('96')) {
```

Comment:
Any reason not to filter ulpfec completely for now? Are we using it in any way?

Reply:
I am not sure, however from my testing this is specific to Google Chrome 96.

I leave this open to be extended in future if needed, as I am not seeing any bugs on Google Chrome 95, 97, Edge (even the one based on Chrome 96) and Opera. 

Maybe the Google engineers screwed up something with their build !?

Reply:
👍

Reply:
Can you please add a comment here (with a link to the Chrome bug report) why this is needed, thanks!

Reply:
@jallamsetty1 done.